<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlsDemoClass</class>
 <widget class="QMainWindow" name="ControlsDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlsDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane{
background:#1e3c6e;border-left:1px solid gray;border-right:1px solid gray;
border-bottom:1px solid gray
}
QTabWidget::tab-bar{
background:#4e6c9e;left:5px;min-height:30px;
}
QTabBar::tab{
background:#2e4c7e;padding:3px;color:white;border: 1px solid #C4C4C3;
border-bottom-color: #1e3c6e; /* same as the pane color */
border-top-left-radius: 4px;
border-top-right-radius: 4px;
min-width: 8ex;min-height: 5ex;
padding: 2px;
}
QTabBar::tab:selected,QTabWidget::tab:hover{
background:#1e3c6e
}
QTabBar::tab:selected{
border-color:#9b9b9b;border-bottom-color:#1e3c6e
}
QTabBar::tab:!selected{
margin-top:2px
}
QTabBar::tab:selected {
margin-left: -4px;
margin-right: -4px;
}
QTabBar::tab:first:selected {
margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}
QTabBar::tab:last:selected{
margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}
QTabBar::tab:only-one {
margin: 0; /* if there is only one tab, we don't want overlapping margins */
}

</string>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Simple Controls</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>50</y>
          <width>556</width>
          <height>62</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>9</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCCheckBox" name="checkBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>10</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCRatioButton" name="radioButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>9</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>treeView</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>tableView</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>listView</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QCListView" name="listView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Input Widgets</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>21</y>
          <width>616</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
         <item row="4" column="2">
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item row="0" column="2">
          <widget class="QFontComboBox" name="fontComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFontComboBox{background:#1e3c6e;color:white;border:2px solid gray;border-radius:3px}
QFontComboBox::editable{background:#23b8ff;}
QFontComboBox::drop-down{border-radius:3px;min-width:30px;border-image:url(:/ControlsDemo/Resources/pics/DownArrow.bmp);}
QFontComboBox::drop-down:pressed{position:relative;top:1px;left:1px}
QFontComboBox::drop-down:on {top: 1px;left: 1px}
QListView{color:white;outline:0px;background: #1e3c6e}
QListView::item{color:white;min-height:30px;border-top:1px solid #63709c;}
QListView::item:selected{color:red;background:#295dab;}
QListView::item:hover{background:#295dab;}
QScrollBar::vertical{min-width:24px;min-height:80px;background:#1e3c6e;margin:24px 0 24px 0;}
QScrollBar::handle:vertical{margin:2px;min-height:50px;background:#295dab;border-radius:5px}
QScrollBar::handle:vertical:hover{background:#125ddb}
QScrollBar::handle:vertical:pressed{background:#125ddb}
QScrollBar::sub-line:vertical{border-image:url(:/ControlsDemo/Resources/pics/listUpArrow.png);height:24px;subcontrol-position:top;subcontrol-origin:margin}
QScrollBar::add-line:vertical{border-image:url(:/ControlsDemo/Resources/pics/listDownArrow.png);height:24px;subcontrol-position:bottom;subcontrol-origin:margin}
QScrollBar::sub-page:vertical{background:#1e3c6e;}
QScrollBar::add-page:vertical{background:#1e3c6e;}
QScrollBar::horizontal{min-width:50px;min-height:24px;background:#1e3c6e;margin:0px 24px 0 24px;}
QScrollBar::handle:horizontal{margin:2px;height:24px;background:#295dabaa;border-radius:5px}
QScrollBar::handle:horizontal:hover{background:#295dab88}
QScrollBar::handle:horizontal:pressed{background:#295dabaa}
QScrollBar::sub-line:horizontal{border-image:url(:/ControlsDemo/Resources/pics/listLeftArrow.png);width:24px;subcontrol-position:left;subcontrol-origin:margin}
QScrollBar::add-line:horizontal{border-image:url(:/ControlsDemo/Resources/pics/listRightArrow.png);width:24px;subcontrol-position:right;subcontrol-origin:margin}
QScrollBar::sub-page:horizontal{background:#1e3c6e;}
QScrollBar::add-page:horizontal{background:#1e3c6e;}
</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCComboBox" name="comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QSpinBox" name="spinBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox{font:bold 19px;background:#1e3c6e;color:white;border:2px solid gray;border-radius:3px;padding-right:15px;selection-background-color: red;}
QSpinBox::down-button{border-radius:3px;min-width:30px;border-image:url(:/ControlsDemo/Resources/pics/DownArrow.bmp);}
QSpinBox::down-button:pressed{position:relative;top:1px;left:1px}
QSpinBox::up-button{border-radius:3px;min-width:30px;border-image:url(:/ControlsDemo/Resources/pics/UpArrow.bmp);}
QSpinBox::up-button:pressed{position:relative;top:1px;left:1px}
</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{font:bold 19px;background:#1e3c6e;color:white;border:2px solid gray;border-radius:3px;}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QCDial" name="dial">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/ControlsDemo/Resources/pics/dialBg.png);
image: url(:/ControlsDemo/Resources/pics/dial.png);
background-image: url(:/ControlsDemo/Resources/pics/dial.png);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1177</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCButton</class>
   <extends>QPushButton</extends>
   <header>qcbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCRatioButton</class>
   <extends>QRadioButton</extends>
   <header>qcratiobutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCListView</class>
   <extends>QListView</extends>
   <header>qclistview.h</header>
  </customwidget>
  <customwidget>
   <class>QCComboBox</class>
   <extends>QComboBox</extends>
   <header>qccombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QCCheckBox</class>
   <extends>QCheckBox</extends>
   <header>qccheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>QCTreeView</class>
   <extends>QTreeView</extends>
   <header>qctreeview.h</header>
  </customwidget>
  <customwidget>
   <class>QCTableView</class>
   <extends>QTableView</extends>
   <header>qctableview.h</header>
  </customwidget>
  <customwidget>
   <class>QCDial</class>
   <extends>QDial</extends>
   <header>qcdial.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ControlsDemo.qrc"/>
 </resources>
 <connections/>
</ui>
